release_notes/ocp-4-7-release-notes.adoc

```diff
@@ -105,6 +105,10 @@ The {product-title} internal registry and image streams now support Open Contain
 [id="ocp-4-7-machine-api"]
 === Machine API
 
+[id="ocp-4-7-machine-api-wait-upgrade"]
+==== Updates are blocked if a worker machine config pool is degraded
+
+If a machine config pool (MCP) is in a `degraded` state, you are blocked from performing an update within a minor version, for example, from 4.7 to 4.8, until all machine config pools are healthy. For z-stream releases, for example, from 4.7.1 to 4.7.2, the upgrade is not blocked. Previously, when updating with a degraded MCP, the update would proceed, but fail when updating the Machine Config Operator (MCO).    
```

Comment:
If a machine config pool is in a `degraded state, Upgradeable is set to False in the cluster operator and you are blocked....